refiner/app/api/v1/demo.py

```diff
@@ -320,9 +334,25 @@ async def demo_upload(
                         "sections_processed": "All sections scoped to condition codes",
                         "method": "ProcessedGrouper-based filtering",
                     },
-                },
+                }
             )
 
+        # ✅ Zip all condition files + eICR file into one archive
+        full_zip_output_path = _create_zipfile_output_directory(refined_zip_output_dir)
+
+        # Add eICR + RR file as well
+        refined_files_to_zip.append(("CDA_EICR.xml", xml_files.eicr))
```

Comment:
I assume we should output the typical casing.
```suggestion
        refined_files_to_zip.append(("CDA_eICR.xml", xml_files.eicr))
```

Reply:
yeah i think we should change the `E` to `e` as well